### ISSUE TITLE
(Feature request) Images should be removed from the HTML export navigation tree

### ISSUE THREAD

**Issue by Mirwaisse** [zadam/trilium#2559]
### Describe feature

Even though images are hidden (by default) in the Trilium note tree, they are still visible in the navigation tree of the HTML export.

![image](https://user-images.githubusercontent.com/25323123/149620615-046fa5af-60d2-4163-baef-71a25b6de10e.png)

**Comment by Mirwaisse** [comment]
Wait, did it get fixed by this commit earlier today? https://github.com/zadam/trilium/commit/c8884f19177e74f3323a6bd679de71cd1a2473d7

**Comment by zadam** [comment]
Hi, the navigation in the export is provided as a quick and easy low-hanging fruit. It's not supposed to be at feature parity with the main UI.

I'm curious, what do you use the navigation for?

> Wait, did it get fixed by this commit earlier today? c8884f1

No, that's a different fix.

**Comment by Mirwaisse** [comment]
> > Wait, did it get fixed by this commit earlier today? [c8884f1](https://github.com/zadam/trilium/commit/c8884f19177e74f3323a6bd679de71cd1a2473d7)
> 
> No, that's a different fix.

Right, I read too fast 😢 

> Hi, the navigation in the export is provided as a quick and easy low-hanging fruit. It's not supposed to be at feature parity with the main UI.

I understand, but I really think that images not included in a note should also be excluded from the navigation tree in the HTML export. It only clutters it...

> I'm curious, what do you use the navigation for?

I share some of my notes on my website using the HTML export feature, for a long time now (well, I used to but not right now since I took my website down (I will get it back up again in the near future)).

I know that since 0.49.X, there is a "share feature" but I do not use a server installation of Trilium. I prefer to use a fully local desktop installation, and share my notes by using the HTML exported pages and host them.

I currently use a script to remove the images from the navigation tree and also add MathJax to the HTML pages of the notes that use LaTeX, but I think it would be better if it was done directly by Trilium.
